tests/drivers/i2s/i2s_speed/src/main.c

```diff
@@ -8,4 +8,7 @@
 #include <zephyr/kernel.h>
 #include <zephyr/ztest.h>
 
-ZTEST_SUITE(drivers_i2s_speed, NULL, NULL, NULL, NULL, NULL);
+extern void *test_i2s_speed_configure(void);
+extern void *test_i2s_speed_rxtx_configure(void);
+ZTEST_SUITE(drivers_i2s_speed, NULL, test_i2s_speed_configure, NULL, NULL, NULL);
+ZTEST_SUITE(drivers_i2s_speed_both_rxtx, NULL, test_i2s_speed_rxtx_configure, NULL, NULL, NULL);
```

Comment:
Why not register these test suites directly in `test_i2s_speed.c` and avoid the above declarations of `_configure` functions?

Reply:
Thank you, I have combined the two into one file, but have combined them into main.c instead.

Reply:
> Thank you, I have combined the two into one file, but have combined them into main.c instead.

Please use `test_i2s_speed.c` as the name of the combined file (there is no requirement to have `main.c`). Diff in your commit will be shorter and it will be easier to check history of particular lines of that file.